src/worker/requirements.txt

```diff
@@ -10,25 +10,27 @@ cachetools==5.3.1
 certifi==2023.7.22
 cffi==1.15.1
 charset-normalizer==3.2.0
+click==8.1.7
 cryptography==41.0.3
 decorator==5.1.1
+deprecation==2.1.0
 dnspython==2.4.1
 environs==9.5.0
 exceptiongroup==1.1.2
 frozenlist==1.4.0
 googleapis-common-protos==1.60.0
 greenlet==3.0.0
 grpc-gateway-protoc-gen-openapiv2==0.1.0
-grpcio
-grpcio-tools
+grpcio==1.56.0
+grpcio-tools==1.48.2
 h11==0.14.0
 h2==4.1.0
 hpack==4.0.0
 httpcore==0.17.3
 httpx==0.24.1
 hyperframe==6.0.1
 idna==3.4
-lancedb
+lancedb==0.3.1
```

Comment:
same thing here with `lancedb` - please remove the version numbers to avoid a dependency clash